browser-chat/Cargo.toml

```diff
@@ -4,6 +4,6 @@ members = ["shared", "cli", "browser-wasm"]
 
 [workspace.dependencies]
 # we define iroh dependencies here to make upgrading easier.
-iroh = { version = "0.34", default-features = false }
-iroh-base = { version = "0.34", default-features = false, features = ["ticket"] }
-iroh-gossip = { version = "0.34", default-features = false, features = ["net"] }
+iroh = { version = "0.35", default-features = false }
+iroh-base = { version = "0.35", default-features = false, features = ["ticket"] }
+iroh-gossip = { version = "0.35", default-features = false, features = ["net"] }
```

Comment:
Have you intentionally kept these at 0.35?
(also in other places. I'll keep commenting on all of them as a pseudo todo list)